.github/workflows/ci.yml

```diff
@@ -48,3 +48,21 @@ jobs:
         dotnet restore -p:Portability=True DotNetLightning.sln
         msbuild src/DotNetLightning.Core/DotNetLightning.Core.fsproj -p:Portability=True -p:TargetFramework=netstandard2.0
 
+  style:
+    needs: build_and_test
```

Comment:
I think this should run before testing.
Since it is more likely to bump in, and it is easier to fix than unit tests.

Reply:
Mmm, are you sure? I actually thought about the order before, and I've always decided to put style the last, because we don't want to annoy the contributor about style fixes until he has got the other (most) important things right.

Reply:
hmm, I see. Well, then run those 2 in parallel? (that is, drop `needs`)

Reply:
> (that is, drop needs)

Ok sure I can do that; but this doesn't remove the "annoying the contributor too early" part :)

Reply:
contributor gets annoyed by CI anyway. Thats what CI is for.

Reply:
Done.